### PR TITLE
Add PoDoFo, muparser and libHaru

### PR DESCRIPTION
**Pull request by Dalan94**
This is new PKGBUILD for [PoDoFo](http://podofo.sourceforge.net/), [muparser](http://muparser.beltoforion.de/) and [libHaru](http://libharu.org/).

It's my firsts PKGBUILD for MSYS2 so they can have several mistakes...

### CODE REVIEW

File: mingw-w64-libharu/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Maintainer: Rémi BERTHO <remi.bertho@openmailbox.org>
+
+_realname=libharu
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
```
Comment (Alexpux) [inline mingw-w64-libharu/PKGBUILD]: Add:
pkgbase=mingw-w64-${_realname}
before this line in all PKGBUILD's

File: mingw-w64-libharu/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+pkgver=2.3.0
+pkgrel=1
+pkgdesc="C library for generating PDF documents"
```
Comment (Alexpux) [inline mingw-w64-libharu/PKGBUILD]: Add "(mingw-w64)" at the end of description in all PKGBUILD's

File: mingw-w64-libharu/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+url="https://github.com/libharu/libharu"
+arch=('i686' 'x86_64')
```
Comment (Alexpux) [inline mingw-w64-libharu/PKGBUILD]: arch=('any') for all mingw packages.

File: mingw-w64-libharu/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+prepare() {
+  cd "$srcdir/$_realname-RELEASE_${pkgver//./_}"
+  patch -p1 -i "$srcdir/win_various_install_files.patch"
+  patch -p1 -i "$srcdir/win_lib.patch"
+}
+
+build() {
+  cd "$srcdir/$_realname-RELEASE_${pkgver//./_}"
+  mkdir -p build
```
Comment (Alexpux) [inline mingw-w64-libharu/PKGBUILD]: this is good to build every arch in different folder. for example: build-${MINGW_CHOST}

File: mingw-w64-libharu/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  cd build
+  MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
+  ${MINGW_PREFIX}/bin/cmake.exe -G"MSYS Makefiles" \
+    -DCMAKE_C_COMPILER=${MINGW_PREFIX}/bin/gcc.exe \
+    -DCMAKE_MAKE_PROGRAM=${MINGW_PREFIX}/bin/mingw32-make.exe \
```
Comment (Alexpux) [inline mingw-w64-libharu/PKGBUILD]: don't need this

File: mingw-w64-libharu/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  patch -p1 -i "$srcdir/win_lib.patch"
+}
+
+build() {
+  cd "$srcdir/$_realname-RELEASE_${pkgver//./_}"
+  mkdir -p build
+  cd build
+  MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
+  ${MINGW_PREFIX}/bin/cmake.exe -G"MSYS Makefiles" \
+    -DCMAKE_C_COMPILER=${MINGW_PREFIX}/bin/gcc.exe \
```
Comment (Alexpux) [inline mingw-w64-libharu/PKGBUILD]: don't need this

File: mingw-w64-libharu/PKGBUILD [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    -DCMAKE_MAKE_PROGRAM=${MINGW_PREFIX}/bin/mingw32-make.exe \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_SHARED_LIBS=ON \
+    -DBUILD_STATIC_LIBS=ON \
+    -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} ..
+  mingw32-make
```
Comment (Alexpux) [inline mingw-w64-libharu/PKGBUILD]: Use "make" not "mingw32-make"

File: mingw-w64-libharu/PKGBUILD [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+package() {
+  cd "$srcdir/$_realname-RELEASE_${pkgver//./_}/build"
+  mingw32-make DESTDIR=$pkgdir install
```
Comment (Alexpux) [inline mingw-w64-libharu/PKGBUILD]: use "make"

**Comment by Dalan94** [comment]
OK I think I fixed all your reports.

**Pull request merged**